engine/title.asm

```diff
@@ -3,17 +3,654 @@ INCLUDE "constants.asm"
 SECTION "Title screen", ROMX[$5D8C], BANK[$01]
 
 IntroSequence:: ; 5d8c
-    ; TODO
+	ld hl, GameFreakLogo ;E4000
+	ld a, $39
+	call FarCall_hl
```

Comment:
Use `callfar`/`callab` (if it's ever added).
```
    callfar GameFreakLogo
```

Reply:
Changed, works. Seems like it was added?